src/condor_starter.V6.1/filter.cpp

```diff
@@ -4,105 +4,89 @@
 #include "classad/classad.h"
 #include "filter.h"
 #include "compat_classad.h"
+#include "compat_classad_util.h"
 
 #include <map>
 #include <string>
 #include <vector>
-#if defined(CXX_TWENTY_IS_REAL_TO_ME)
 #include <algorithm>
 #include <ranges>
-#endif   /* CXX_TWENTY_IS_REAL_TO_ME */
 
-//
-// Excepting the special case, these two function are identical, so
-// they should be refactored together.  Maybe by having a callback
-// for each CLASSAD_VALUE (with the extracted ClassAd and holding
-// attribute name as arguments)?
-//
-// ... would probably be less ugly as a two-pass algorithm in the
-// top-level wrapper.
-//
 
+typedef std::map<std::string, classad::Value::ValueType, classad::CaseIgnLTStr> Filter;
```

Comment:
I'm leaning towards the `using existing = new_type` syntax more and more over typedef, fwiw

Reply:
What's the technical difference?  (If it's not just syntax?)

Reply:
Just syntax.